### ISSUE TITLE
Support for Equipment Information

### ISSUE THREAD

**Issue by dacite** [Gabouchet/wot-game-reader#1]
This information seems to be present in `source/res/scripts/item_defs/vehicles/common/optional_devices.xml`. I can work on this if you'd like.

**Comment by gabrielhamel** [comment]
Oh sorry I didn't see your comment. Seems like a cool feature. I will check this tomorrow.
Btw if you have some advices to give me on architecture of the project. I'm completely open =)

**Comment by dacite** [comment]
I think its great so far! I've already used this crate to extract map bounding boxes and it was very easy to use.

**Comment by gabrielhamel** [comment]
Yeah, that was my goal. I made a discord bot that take a replay an make a gif of the minimap.

![](https://cdn.discordapp.com/attachments/815352807388741642/1067987127574462534/file.jpg)

Pretty funny

**Comment by dacite** [comment]
That is cool! link to bot?

I did something similar (draw tank positions) using this crate but a website.

**Comment by gabrielhamel** [comment]
https://github.com/Gabouchet/wot-discord-bot
https://discord.com/api/oauth2/authorize?client_id=1065025428068237452&permissions=0&scope=bot

But I didn't finished the integration with this crate. I will refactor many things and adding up a database to handle bot interactions.

**Comment by dacite** [comment]
Thank you for sharing!
